testing/test_cacheREADME.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import, division, print_function
+import _pytest
+
+pytest_plugins = ("pytester",)
+
+
+class Helper(object):
+
+    def check_readme(self, testdir):
+        config = testdir.parseconfigure()
+        readme = config.cache._cachedir.join("README.md")
+        return readme.isfile()
+
+
+class TestReadme(Helper):
```

Comment:
Let's move this class to `test_cache.py`, I don't think we need to keep this in a separate file just because of a single test.

Reply:
will do.